docker/php/www.conf

```diff
@@ -440,4 +440,17 @@ pm.max_spare_servers = 3
 
 php_admin_flag[log_errors] = on
 
-php_admin_value[date.timezone] = Europe/Paris
\ No newline at end of file
+php_admin_value[date.timezone] = Europe/Paris
+
+;Debug Settings
+php_admin_value[xdebug.mode] = develop
+php_admin_value[debug.start_with_request] = yes
+php_admin_value[xdebug.log]="/tmp/xdebug.log"
```

Comment:
Shouldn't we redirect the output to stdout/stderr since we are inside a Docker container?

I'm not familiar with xdebug so I don't know whether it make sense.

Reply:
Why not, I've change it